app/views/users/new.html.haml

```diff
@@ -1,42 +1,39 @@
+%head
+  %script(type="text/javascript" src="https://js.stripe.com/v2/")
+  %script(type="text/javascript")
+    Stripe.setPublishableKey('pk_test_0oQRDCrrnEWxCN7PoAhYJ2aL');
+  = javascript_include_tag "payment"
+
+
 %section.register.container
   .row
     .col-sm-10.col-sm-offset-1
-      .form-horizontal
-        = bootstrap_form_for @user, method: 'post' do |f|
-          %header
-            %h1 Register
-          %fieldset
-            .form-group
-              .control-label.col-sm-2= f.label :email, 'Email Address'
-              .col-sm-6
-                = f.email_field :email, hide_label: true, icon: 'envelope'
-            .form-group
-              .control-label.col-sm-2= f.label :password, 'Password'
-              .col-sm-6
-                = f.password_field :password, hide_label: true, icon: 'lock'
-            .form-group
-              .control-label.col-sm-2= f.label :name, 'Full Name'
-              .col-sm-6
-                = f.text_field :name, hide_label: true, icon: 'user'
-            - if @invitation
-              = hidden_field_tag :invitation_token, @invitation.token
-            .col-sm-offset-2.controls
-              = f.submit 'Sign Up', class: 'btn btn-default'        
-        / %fieldset.credit_car
-        /   .form-group
-        /     %label.control-label.col-sm-2 Credit Card Number
-        /     .col-sm-6
-        /       %input.form-control(type="text")
-        /   .form-group
-        /     %label.control-label.col-sm-2 Security Code
-        /     .col-sm-6
-        /       %input.form-control(type="text")
-        /   .form-group
-        /     %label.control-label.col-sm-2 Expiration
-        /     .col-sm-3
-        /       = select_month(Date.today, {add_month_numbers: true},  class: 'form-control')
-        /     .col-sm-2
-        /       = select_year(Date.today.year, {start_year: Date.today.year, end_year: Date.today.year + 4}, class: 'form-control')
-        / %fieldset.actions.control-group.col-sm-offset-2
-        /   .controls
-        /     %input(type="submit" value="Sign Up" class="btn btn-default")
+      = bootstrap_form_for @user, layout: :horizontal, control_col: "col-sm-6", method: 'post', html: {id: 'payment-form'} do |f|
+        %header
+          %h1 Register
+        %fieldset
+          = f.email_field :email, icon: 'envelope'
+          = f.password_field :password, icon: 'lock'
+          = f.text_field :name, icon: 'user'
+          - if @invitation
+            = hidden_field_tag :invitation_token, @invitation.token                 
+        %fieldset.credit_card
+          .form-group
+            %span.payment-errors.alert.alert-danger{style: "display:none;"}
```

Comment:
Rather than inline css, you should use a class that would make it `display: none`.